packages/fluentui/react/src/components/Card/CardPreview.tsx

```diff
@@ -0,0 +1,100 @@
+import { Accessibility, cardPreviewBehavior, CardPreviewBehaviorProps } from '@fluentui/accessibility';
+import { getElementType, getUnhandledProps, useAccessibility, useStyles, useTelemetry } from '@fluentui/react-bindings';
+import * as _ from 'lodash';
+import * as PropTypes from 'prop-types';
+import * as React from 'react';
+// @ts-ignore
+import { ThemeContext } from 'react-fela';
+import { FluentComponentStaticProps, ProviderContextPrepared, WithAsProp, withSafeTypeForAs } from '../../types';
+import { ChildrenComponentProps, commonPropTypes, createShorthandFactory, UIComponentProps } from '../../utils';
+
+export interface CardPreviewProps extends UIComponentProps, ChildrenComponentProps {
+  /**
+   * Accessibility behavior if overridden by the user.
+   * */
+  accessibility?: Accessibility<CardPreviewBehaviorProps>;
+
+  /**
+   *
+   * If preview is in horizontal card
+   */
+
+  horizontal?: boolean;
+
+  /**
+   * Do not add margin after the preview
+   */
+  noMarginAfter?: boolean;
+}
+
+export type CardPreviewStylesProps = Pick<CardPreviewProps, 'horizontal' | 'noMarginAfter'>;
+
+const CardPreview: React.FC<WithAsProp<CardPreviewProps>> & FluentComponentStaticProps<CardPreviewProps> = props => {
+  const context: ProviderContextPrepared = React.useContext(ThemeContext);
+  const { setStart, setEnd } = useTelemetry(CardPreview.displayName, context.telemetry);
+  setStart();
+
+  const { className, design, styles, variables, children, horizontal, noMarginAfter } = props;
+  const ElementType = getElementType(props);
+  const unhandledProps = getUnhandledProps(CardPreview.handledProps, props);
+  const getA11yProps = useAccessibility(props.accessibility, {
+    debugName: CardPreview.displayName,
+    rtl: context.rtl
+  });
+
+  const { classes } = useStyles<CardPreviewStylesProps>(CardPreview.displayName, {
+    className: CardPreview.className,
+    mapPropsToInlineStyles: () => ({
+      className,
+      design,
+      styles,
+      variables,
+      horizontal,
+      noMarginAfter
```

Comment:
```suggestion
    mapPropsToStyles: () => ({ horizontal, noMarginAfter }),
    mapPropsToInlineStyles: () => ({
      className,
      design,
      styles,
      variables,
```

Reply:
I noticed other people being confused by this separator too. Maybe we should consider changing the API here a bit.